### ISSUE TITLE
Support configuration of package structure depending on language

### ISSUE THREAD

**Issue by sebastianhaberey** [btc-ag/service-idl#71]
Had a talk with @GerrietReents about this a couple days ago. Since the existing package structure may well be different depending on the language, it would be nice to have some sort of configuration file or option where you could specify packages per language.

**Comment by sigiesec** [comment]
I agree that this should be made more flexible. However, supporting arbitrary package structures just by configuration would probably be very complex to implement and to specify.

If you have specific wishes/requirements, it would be good to know them. Can you describe what package structure you would like to have?

**Comment by sigiesec** [comment]
NB: As you already said, this is quite different for each target language/technology, and it must also be implemented separately for each of those. So please make suggestions specifically for each target technology. Maybe also split this up into separate issues for each target technology, so they can be tracked easier.
